Extending/Property-Editors/value-converters.md

```diff
@@ -25,9 +25,9 @@ Implement `IPropertyValueConverter` from the `Umbraco.Core` namespace on your cl
 
 ### bool IsConverter(PublishedPropertyType propertyType) ###
 
-This method is called for each PublishedPropertyType (document type property) at application startup. By returning True your value converter will be registered for that property type and your conversion methods will be executed when ever that value is requested. 
+This method is called for each PublishedPropertyType (document type property) at application startup. By returning `True` your value converter will be registered for that property type and your conversion methods will be executed when ever that value is requested. 
 
-Example, checking if the PublishedPropertyType PropertyEditorAlias property is equal to the alias of the core content editor
+Example: Checking if the PublishedPropertyType PropertyEditorAlias property is equal to the alias of the core content editor
```

Comment:
I am not native English, but adding a upper case after a : feels odd.

Reply:
To my knowledge (and eyes too), unless what follows the colon is a list of things, the first world should be uppercased as in a new sentence. The way I read it is like this: "Here's an example: Checking if [...]".

Reply:
Well it actually depends on British English or American English, https://www.grammarly.com/blog/capitalization-after-colons/. So as the Umbraco Docs are mainly American English, lets go with the capitalization (see the z)